internal/rest/mgmt/helpers.go

```diff
@@ -77,7 +87,157 @@ func ServicePolicyFromFilter(client *rest_management_api_client.ZitiEdgeManageme
 	params.SetTimeout(5 * time.Second)
 	resp, err := client.ServicePolicy.ListServicePolicies(params, nil)
 	if err != nil {
-		log.Errorf("Could not obtain an ID for the service with filter %s: %v", filter, err)
+		log.Errorf("Could not obtain an ID for the service policy with filter %s: %v", filter, err)
+		return nil
+	}
+	if resp == nil || resp.Payload == nil || resp.Payload.Data == nil || len(resp.Payload.Data) == 0 {
```

Comment:
```
if resp == nil || resp.Payload == nil || resp.Payload.Data == nil || len(resp.Payload.Data) == 0 {
```
is repeated verbatim a few times (10). Feels like a candidate to make a small validator function to me